script/generate-backups

```diff
@@ -65,6 +65,16 @@ main() {
     add_redis_resource
     backup_cluster "${out}"
   done
+
+  # generate backup without MySQL and MongoDB, and with STATUS_KEY missing from the dashboard
+  local version="v20160814.0"
+  bootstrap_cluster "${version}"
+  git_deploy_app
+  add_redis_resource
+  remove_mysql
+  remove_mongodb
+  remove_dashboard_status_key
```

Comment:
Rather than modifying the latest release, we should use a version where these cases were true.

Version `v20160309.0` (included above) satisfies all of these cases (MariaDB was added in `v20160311.1`, MongoDB in `v20160706.0` and dashboard `STATUS_KEY` in `v20160620.1`).